back-end-side/Controllers/Sorting.cs

```diff
@@ -10,7 +11,7 @@ public class Sorting
         static private String[] HouseMaintenanceShops = { "JYSK", "MOKI VEZI", "SENUKAI" };
         static private String[] EntertainmentShops = { "GG ARENA" };
 
-        public double foodSum, clothesSum, carSum, houseSum, entertainmentSum, otherSum;
+        //public double foodSum, clothesSum, carSum, houseSum, entertainmentSum, otherSum;
```

Comment:
Are they really no longer needed for printing the statistics?

Reply:
Yes, they are not needed